readme.md

```diff
@@ -29,7 +29,24 @@ $ npm install --save dom-chef
 
 ## Usage
 
-Make sure to use a JSX transpiler and set JSX pragma to `h`.
+Make sure to use a JSX transpiler, set JSX pragma to `h` and optinally the 
+pragmaFrag to `DocumentFragment` [if you need fragment support](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html).
```

Comment:
`pragmaFrag` should be wrapped in backticks too and should link to https://babeljs.io/docs/en/next/babel-plugin-transform-react-jsx.html#pragmafrag